scripts/update-cdn-urls.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Updates the CDN URLs in the README.md to the latest major version. Does
+ * nothing if the README.md is already up-to-date.
+ *
+ * It assumes the major version in the README is either equal to the current
+ * package major version, or the package minor version minus 1. If not the
+ * behaviour of this script should be consider undefined.
+ */
+
+const fs = require("fs");
+const path = require("path");
+const getMajorVersion = require("semver/functions/major");
+
+const rootDir = path.resolve(__dirname, "..");
+const packageFileJson = path.resolve(rootDir, "package.json");
```

Comment:
It seems that the name should be `packageJsonFile`, as has been named in #5546:

```suggestion
const packageJsonFile = path.resolve(rootDir, "package.json");
```